validphys2/src/validphys/renametools.py

```diff
@@ -12,6 +12,10 @@ class Spinner:
     """ Context manager to provide a spinning cursor
     while validphys performs some other task silently.
 
+    When exececuted in a TTY, it shows a spinning cursor for the duration of
```

Comment:
What's TTY?

Reply:
Uh https://www.howtogeek.com/428174/what-is-a-tty-on-linux-and-how-to-use-the-tty-command/